### ISSUE TITLE
DEP: distutils deprecated

### ISSUE THREAD

**Issue by snowman2** [geopandas/geopandas#2083]
https://www.python.org/dev/peps/pep-0632/

**Comment by danschef** [comment]
Python 3.10 now raises a DeprecationWarning:

```python
/root/mambaforge/envs/ci_env/lib/python3.10/site-packages/geopandas/_compat.py:227
distutils Version classes are deprecated. Use packaging.version instead.
```

Looks like this is already implemented in https://github.com/geopandas/geopandas/pull/2084 but not merged yet.

**Comment by martinfleis** [comment]
@danschef thanks for the nudge. I'll make sure it is in ahead of the next release.

**Comment by stefaneidelloth** [comment]
In my tests I get still deprecation warnings using geopandas 0.10.2:

```
======================================================================================================== warnings summary ======================================================================================================== 
..\..\App\WinPython\python-3.10.2.amd64\lib\site-packages\geopandas\_compat.py:18
  C:\python_env\App\WinPython\python-3.10.2.amd64\lib\site-packages\geopandas\_compat.py:18: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    PANDAS_GE_10 = str(pd.__version__) >= LooseVersion("1.0.0")



..\..\App\WinPython\python-3.10.2.amd64\lib\site-packages\geopandas\_compat.py:19
  C:\python_env\App\WinPython\python-3.10.2.amd64\lib\site-packages\geopandas\_compat.py:19: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    PANDAS_GE_11 = str(pd.__version__) >= LooseVersion("1.1.0")

..\..\App\WinPython\python-3.10.2.amd64\lib\site-packages\geopandas\_compat.py:20
  C:\python_env\App\WinPython\python-3.10.2.amd64\lib\site-packages\geopandas\_compat.py:20: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    PANDAS_GE_115 = str(pd.__version__) >= LooseVersion("1.1.5")

..\..\App\WinPython\python-3.10.2.amd64\lib\site-packages\geopandas\_compat.py:21
  C:\python_env\App\WinPython\python-3.10.2.amd64\lib\site-packages\geopandas\_compat.py:21: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    PANDAS_GE_12 = str(pd.__version__) >= LooseVersion("1.2.0")

..\..\App\WinPython\python-3.10.2.amd64\lib\site-packages\geopandas\_compat.py:29
  C:\python_env\App\WinPython\python-3.10.2.amd64\lib\site-packages\geopandas\_compat.py:29: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    SHAPELY_GE_17 = str(shapely.__version__) >= LooseVersion("1.7.0")

..\..\App\WinPython\python-3.10.2.amd64\lib\site-packages\geopandas\_compat.py:30
  C:\python_env\App\WinPython\python-3.10.2.amd64\lib\site-packages\geopandas\_compat.py:30: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    SHAPELY_GE_18 = str(shapely.__version__) >= LooseVersion("1.8")

..\..\App\WinPython\python-3.10.2.amd64\lib\site-packages\geopandas\_compat.py:31
  C:\python_env\App\WinPython\python-3.10.2.amd64\lib\site-packages\geopandas\_compat.py:31: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    SHAPELY_GE_20 = str(shapely.__version__) >= LooseVersion("2.0")

..\..\App\WinPython\python-3.10.2.amd64\lib\site-packages\geopandas\_compat.py:227
..\..\App\WinPython\python-3.10.2.amd64\lib\site-packages\geopandas\_compat.py:227
  C:\python_env\App\WinPython\python-3.10.2.amd64\lib\site-packages\geopandas\_compat.py:227: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    PYPROJ_LT_3 = LooseVersion(pyproj.__version__) < LooseVersion("3")

..\..\App\WinPython\python-3.10.2.amd64\lib\site-packages\geopandas\_compat.py:228
..\..\App\WinPython\python-3.10.2.amd64\lib\site-packages\geopandas\_compat.py:228
  C:\python_env\App\WinPython\python-3.10.2.amd64\lib\site-packages\geopandas\_compat.py:228: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    PYPROJ_GE_31 = LooseVersion(pyproj.__version__) >= LooseVersion("3.1")

```

**Comment by felix-ht** [comment]
@martinfleis whats the timeline on the release?

**Comment by martinfleis** [comment]
It was planned for January. I think that says it all :D. No idea to be honest. Hopefully soon.

**Comment by nmoreaud** [comment]
Which release?

**Comment by martinfleis** [comment]
@nmoreaud the fix for this was part of the 0.11 release.
